source/common/common/utility.cc

```diff
@@ -20,8 +22,72 @@
 #include "spdlog/spdlog.h"
 
 namespace Envoy {
+
+namespace {
+
+class SubsecondConstantValues {
+public:
+  const size_t DEFAULT_SECONDS_LENGTH{10};
```

Comment:
Nit: I don't think these scalars need to be part of the singleton.

Reply:
Done